notebooks/mlruns/0/meta.yaml

```diff
@@ -0,0 +1,6 @@
+artifact_location: file:///home/jr/learning/databricks_mlops/marvelous-databricks-course-JuanOnOne/notebooks/mlruns/0
+creation_time: 1730377060300
+experiment_id: '0'
```

Comment:
A suggestion: add mlruns to .gitignore file and ignore these. mlruns can contain sensitive information and it's better to not to add them to version control